dipy/reconst/dti.py

```diff
@@ -1770,7 +1770,7 @@ def restore_fit_tensor(design_matrix, data, sigma=None, jac=True,
 
     References
     ----------
-    Chang, L-C, Jones, DK and Pierpaoli, C (2005). RESTORE: robust estimation
+    [1] Chang, L-C, Jones, DK and Pierpaoli, C (2005). RESTORE: robust estimation
```

Comment:
I believe this should be `.. [1]` for this to be properly linked and rendered by sphinx.

Reply:
@arokem yes, you're right. Custom anchors should be created for linking the references. They were missing for some other references too.